pkg/controllers/updaterun/execution_integration_test.go

```diff
@@ -1304,6 +1305,122 @@ var _ = Describe("UpdateRun execution tests - single stage", func() {
 			}, timeout, interval).Should(BeTrue(), "failed to ensure the approvalRequest is not recreated")
 		})
 	})
+
+	Context("Cluster staged update run should update clusters one by one - different states (Initialized -> Execute)", Ordered, func() {
+		var wantMetrics []*io_prometheus_client.Metric
+		BeforeAll(func() {
+			By("Creating a new clusterStagedUpdateRun")
+			updateRun.Spec.State = placementv1beta1.StateNotStarted
```

Comment:
Could we also update the var names to match initialize, execute as part of this PR ?